inductiva/utils/output_info.py

```diff
@@ -14,26 +14,33 @@ class FileInfo:
 
 
 @dataclass
-class OutputContents:
+class OutputInfo:
     """Information about the contents of a Task output archive."""
-    size: int
-    contents: List[FileInfo]
+    files: List[FileInfo]
+    total_size: int
+
+    @property
+    def n_files(self) -> int:
+        return len(self.files)
 
     @property
-    def num_files(self) -> int:
-        return len(self.contents)
+    def total_compressed_size(self) -> int:
+        return sum(file.compressed_size for file in self.files)
```

Comment:
these are bytes?
maybe change the name of the method to reflect the units:

```suggestion
        def total_compressed_size_bytes(self):
```